### ISSUE TITLE
JSON error

### ISSUE THREAD

**Issue by picklecillo** [inkdropapp/inkdrop-alfred-workflow#1]
After setting up the local server and credentials, i'm not finding any notes.
Through the debugger, this was logged.

```
...
[01:26:12.965] ERROR: Inkdrop[Script Filter] JSON error: JSON text did not start with array or object and option to allow fragments not set. in JSON:
Warning: Invalid argument supplied for foreach() in /Users/me/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.F742D332-1908-4C42-8946-9B3FB6808D20/search.php on line 25
<?xml version="1.0"?>
<items><item uid="inkdrop" arg="gcp" valid="yes"><arg>gcp</arg><title>No notes found</title><subtitle>No notes found. Search Inkdrop for gcp</subtitle><icon>icon.png</icon></item></items>
```

Is it just me? 😬

**Comment by craftzdog** [comment]
That means it looks that you misconfigured the workflow.
You can see the access log by running Inkdrop from terminal.

**Comment by picklecillo** [comment]
Ok. Tried running from command line. 
Did not get any access logs from the Alfred query, while a simple `curl 127.0.0.1:19840/` did generate a ` xxx GET / 401 1ms -` log.
From this I would guess Alfred is not even getting to the querying the db part. 
Anywhere else I could look?

**Comment by craftzdog** [comment]
Yeah, looks like so.
As you can see, the workflow is just a couple of PHP scripts.
You can tweak https://github.com/inkdropapp/inkdrop-alfred-workflow/blob/main/src/search.php to output variables with `print_r` and `var_dump`, then toggle Alfred's debug mode by clicking the spider button.

**Comment by picklecillo** [comment]
I've just left a PR with the solution I've found for this -> https://github.com/inkdropapp/inkdrop-alfred-workflow/pull/2

**Comment by craftzdog** [comment]
Great! I'll roll it out.

**Comment by craftzdog** [comment]
Landed in [v0.1.1](https://github.com/inkdropapp/inkdrop-alfred-workflow/releases/tag/v0.1.1)🎉
Thanks!
